### DISCUSSION TITLE
marpyter - Marp for JupyterLab

### DISCUSSION THREAD

**Discussion by trungleduc** [marp-team/marp#442] [Show and tell]
Hi all,
I created a JupyterLab extension called `marpyter`. Similar to the Marp VS Code extension, `marpyter` allows users to preview the Marp slides directly from JupyterLab.
You can try it out at https://github.com/trungleduc/marpyter 

![marpyter](https://github.com/marp-team/marp/assets/4451292/6b704304-baac-4ce4-b3ce-d14f0a0dfd21)

**Comment by yhatt** [comment]
That's awesome! Thanks you for extending the ecosystem of Marp :smile:
